lambdas/data-migration2/tests/test-executions.js

```diff
@@ -493,3 +495,22 @@ test.serial('migrateExecutions processes all non-failing records', async (t) =>
   );
   t.is(records.length, 1);
 });
+
+test.serial('migrateExecutions logs summary of migration every 100 records', async (t) => {
+  const logSpy = sinon.spy(Logger.prototype, 'info');
+
+  const executions = [];
+  for (let i = 0; i <= 200; i += 1) {
```

Comment:
If we make the logging interval configurable like I mentioned above, then we don't need to create 200 records for the unit test, which seems perhaps more ideal